src/keys.rs

```diff
@@ -1288,4 +1295,46 @@ mod tests {
             assert_eq!(internal_ovk.0, tv.internal_ovk);
         }
     }
+
+    #[test]
+    fn bip340_test_vector() {
```

Comment:
This test uses the first test vector provided in the BIPs repository (the only one with the all-zero auxiliary randomness we use).